go.mod

```diff
@@ -380,6 +380,7 @@ require (
 	github.com/gsterjov/go-libsecret v0.0.0-20161001094733-a6f4afe4910c // indirect
 	github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed // indirect
 	github.com/hashicorp/cronexpr v1.1.1 // indirect
+	github.com/hashicorp/go-bexpr v0.1.12 // indirect
```

Comment:
nit: Indirect seems weird, does this need a `go mod tidy`?

Reply:
Good call. I re-ran `go mod tidy` and it moved it to a direct dependency.